### DISCUSSION TITLE
Json path options change

### DISCUSSION THREAD

**Discussion by octoshikari** [Netflix/conductor#2765] [Ideas]
Hello everyone! We need ```Option.DEFAULT_PATH_LEAF_TO_NULL``` for json path and temporary we create custom build. But maybe it will be helpful to support in application.properties new setting for optional enable this feature

![image](https://user-images.githubusercontent.com/11928813/153281620-567be09f-a551-4c1e-a298-ee4716af3fce.png)

**Comment by apanicker-nflx** [comment]
Please create a PR with the proposed change. Thanks.

**Reply by octoshikari** [reply]
Yes sure. Thank you for answer
